vnext/Microsoft.ReactNative/Fabric/Composition/ImageComponentView.cpp

```diff
@@ -203,17 +204,33 @@ void ImageComponentView::ensureDrawingSurface() noexcept {
   winrt::check_hresult(m_wicbmp->GetSize(&width, &height));
 
   if (!m_drawingSurface && m_wicbmp) {
+    winrt::Windows::Foundation::Size drawingSurfaceSize{static_cast<float>(width), static_cast<float>(height)};
+
+    const auto imageProps = std::static_pointer_cast<const facebook::react::ImageProps>(m_props);
```

Comment:
Used to be `const auto &imageProps = *std::static_pointer_cast`. Are you making a shift to be more pointer-centric instead of ref-centric to be more clear this isn't creating a copy?